### ISSUE TITLE
Load multiple schematisations in the same QGIS project

### ISSUE THREAD

**Issue by leendertvanwolfswinkel** [nens/threedi-schematisation-editor#186]
Users want to load multiple schematisations in the same qgis project. The reason users want this is to compare schematisations. Not to edit multiple simulations at the same time.

I can imagine it complicates some things, that would be easier if only one is "active" or "editable" or something, and the others are read-only

Comment by @ldebek:
Probably I would need to implement some kind of contex manager to handle switching between multiple schematisations. Shouldn't be super complicated as all layers are wrapped with LayerManager class instance while loading, but it will take a  bit to cover all corner cases, to avoid breaking things. In terms of estimates my :crystal_ball: says ~ 2 days

**Comment by Overmeen** [comment]
The functionality to load in multiple schematisations works nicely. Also, editing schematisations seems to work fine, only the layers in the same schematisation become "editable" when toggling editing. 

I notice that there is a way of control over which schematisations a user is saving, removing and or loading features in, by selecting the layer group or a layer from a group. For now, I think that is good. However, maybe in the future we can find a way with for instance a dropdown menu, in that way the user can actively choose which schematisation is saved or removed etc.. However, I think it would be good to wait on @leendertvanwolfswinkel for his thoughts on this.

A small thing that can be fixed now: I tried to load the same schematisation twice, and it did not work which is of course good. But the error message was empty. The text could be: "This schematisation is already loaded."
<img width="228" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBMzQ3Qmc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--984ac43479090bd01bfc0d66e2e134485461340f/image.png" alt="image.png" />

**Comment by ldebek** [comment]
@Overmeen Warning added.

**Comment by Overmeen** [comment]
@ldebek it only works with "Open 3Di Geopackage", the error message is empty for "Load from spatiallite".

**Comment by ldebek** [comment]
@Overmeen Added fix for the "Load from spatiallite" workflow.

**Comment by leendertvanwolfswinkel** [comment]
I have also tried this and it all seems to work great. 

As @Overmeen also commented, we do need a clearer way to choose one of the loaded schematisations for:
- Save to spatialite
- Remove 3Di model
- Import vector data

I suggest the following logic:
- I only one schematisation is loaded, use the UX logic as we currently have in the production version
- If multiple schematisations are loaded and the user clicks on one of the buttons mentioned above, display a popup window with a dropdown menu from which you can choose which schematisation you want to perform the action on. 
- Items in the combobox can be displayed as the file name ("My schematisation"). When hovering over the item, it should show the full path of the file, e.g. "C:\Users\user.name\3Di\My schematisation\revision 4\schematisation\My schematisation.gpkg"

**Comment by leendertvanwolfswinkel** [comment]
@ldebek I will make a new ticket for the comment above
